prody/proteins/interactions.py

```diff
@@ -3917,19 +3936,47 @@ def getInteractions(self, **kwargs):
     
         :arg selection2: selection string
         :type selection2: str 
+        
+        :arg replace: Used with selection criteria to set the new one
+                      If set to True the selection will be replaced by the new one
+        :type replace: True or False
+                       by default is False
             
         Selection:
         If we want to select interactions for the particular residue or group of residues: 
             selection='chain A and resid 1 to 50'
         If we want to study chain-chain interactions:
             selection='chain A', selection2='chain B'  """
-
+        
+        replace = kwargs.pop('replace', False)
+        
         if len(kwargs) != 0:
             sele_inter = []
             for i in self._interactions_traj:
                 for nr_j,j in enumerate(i):
                     sele_inter.append(filterInteractions(i[nr_j], self._atoms, **kwargs))
+            
+            if replace == True:
+                try:
+                    trajectory = self._traj
+                    numFrames = trajectory._n_csets
+                except:
+                    # If we analyze previously saved PKL file we doesn't have dcd information
```

Comment:
`it doesn't have` not `we doesn't have`

Reply:
Ok. I will change that. I sent you an e-mail with the Jupyter Notebook file for tests (for a single PDB file) to see how those functions work now.
Now, it is very convenient if someone analyzes chain-chain interactions and wants to select only those interactions.